packages/expo/src/polyfills/base64Polyfill.ts

```diff
@@ -4,6 +4,7 @@ if (!global.btoa) {
   global.btoa = encode;
```

Comment:
❓ do you think we should also always use the `encode` polyfill in case there are differences between them and cause issues?

Reply:
i think we should be consistent and use `encode`, just like Stefanos indicated.

Reply:
We can, but I can't see how that would result in an error. This is supposed to be a temporary fix for our customers that are encountering this issue due to the Hermes JS engien. A fix for that is already merged by them.

Reply:
@desiprisg in a rare case that someone is using both `encode` and `decode` they may get issues. These functions work together, they better come from the same library to be extra safe. Better safe than sorry.

Reply:
After thinking more about this, I think @anagstef is correct here. Please ignore my offline comment about this and lets polyfill both